cli.js

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env node
+
+'use strict'
+
+const { RuntimeApiClient } = require('@platformatic/control')
+const { select } = require('@inquirer/prompts')
+const { start } = require('./lib/start')
+
+async function getLocationDetails (client, runtime) {
+  try {
+    // Try to get the runtime config
+    const config = await client.getRuntimeConfig(runtime.pid)
+
+    // Log the config object to help debug what properties are available
+    // console.log('Runtime config:', JSON.stringify(config, null, 2));
+
+    // Check various possible properties where the config path might be stored
+    let configPath = 'Unknown'
+    if (config.path) {
+      configPath = config.path
+    } else if (config.configFile) {
+      configPath = config.configFile
+    } else if (config.server && config.server.path) {
+      configPath = config.server.path
+    } else if (runtime.configPath) {
+      configPath = runtime.configPath
+    }
+
+    return {
+      configPath,
+      cwd: runtime.cwd || 'Unknown'
+    }
+  } catch (error) {
+    // console.error('Error fetching config:', error.message);
+    return {
+      configPath: 'Unknown (config not available)',
+      cwd: runtime.cwd || 'Unknown'
+    }
+  }
+}
+
+async function main () {
+  try {
+    // Get available runtimes
+    const client = new RuntimeApiClient()
+
+    try {
+      const runtimes = await client.getRuntimes()
```

Comment:
Similarly to what we do [here](https://github.com/platformatic/watt-admin/blob/main/web/backend/routes/root.ts#L26), we should filter out the `watt-admin` runtime in case the user has already an instance up and running.

Reply:
Basically, I shouldn't be able to do that from the `cli.js` (it doesn't make sense from a UX perspective)
![image](https://github.com/user-attachments/assets/c0dd9f15-27ad-4a86-abc0-49d45ae7697b)